src/utils/toast.js

```diff
@@ -0,0 +1,18 @@
+const makeToast = (message, type) => {
+  // Create toast element
+  const toastElement = document.createElement("div");
+  toastElement.textContent = message;
+  toastElement.className = `toast ${
+    type === "success" ? "bg-green-500 text-white" : "bg-red-500 text-white"
+  }`;
+
+  // append toast to doc
```

Comment:
we should still be using react-hot-toast behind the scenes. we just want to group all the toasts and have one entry point so we can easily make changes to our toasting